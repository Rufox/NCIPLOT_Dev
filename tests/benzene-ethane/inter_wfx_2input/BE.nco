 # ----------------- NCIPLOT ------------------------
 # --- PLOTTING NON COVALENT INTERACTION REGIONS ----
 # ---             E.R. Johnson                  ----
 # ---          J. Contreras-Garcia              ----
 # ----------    Duke University         ------------
 #                                                   
 # ---             A. de la Roza                  ---
 # --------- University of California Merced --------
 #                                                   
 # ---               R. A. Boto                   ---
 # ---                 C. Quan                     --
 # --------  Université Pierre et Marie Curie -------
 # --------------------------------------------------
 # ---              Please cite                  ----
 # --J. Am. Chem. Soc., 2010, 132 (18), pp 6498–6506-
 # --------------------------------------------------
 # --------------------------------------------------
 # ---     Contributions for the wfn properties  ----
 # ---      from H. L. Schmider are acknowledged  ---
 # --------------------------------------------------
 # --------------------------------------------------
 # ---     Contributions for the wfx reader      ----
 # ---      from Dave Arias are acknowledged      ---
 # --------------------------------------------------
 #
 # Start -- 2020.01.07, 03:48:41.615

File: /home/ruben/Documents/software/nciplot-3.0/src/nciplot/dat/h__lda.wfc
WARNING(readgrid): Atomic density file not found
File: /home/ruben/Documents/software/nciplot-3.0/src/nciplot/dat/c__lda.wfc
WARNING(readgrid): Atomic density file not found
-----------------------------------------------------
      INPUT INFORMATION:
-----------------------------------------------------

      MIND YOU
      ONLY ANALYZING INTERMOLECULAR INTERACTIONS     


-----------------------------------------------------
      Calculation details:
-----------------------------------------------------
RHO  THRESHOLD   (au): 0.50
RDG  THRESHOLD   (au): 1.00
DISCARDING RHO PARAM : 0.95

-----------------------------------------------------
      Writing output in the following units:
-----------------------------------------------------

 Sign(lambda2)xDensity x Reduced Density Gradient    = BE.dat

 Reduced Density Gradient,RDG      = BE-grad.cube
 Sign(lambda2)xDensity,LS          = BE-dens.cube
 VMD script                        = BE.vmd



-----------------------------------------------------
      Operating grid and increments: Grid-1
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  1.13    1.13    1.13
 nx,ny,nz  =   11      11       9

 Time for computing density & RDG =   0.05 secs
  0.00% of small boxes are removed.


-----------------------------------------------------
      Operating grid and increments: Grid-2
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  0.76    0.76    0.76
 nx,ny,nz  =   17      16      13

 Time for computing density & RDG =   0.09 secs
  3.75% of small boxes are removed.


-----------------------------------------------------
      Operating grid and increments: Grid-3
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  0.38    0.38    0.38
 nx,ny,nz  =   33      31      26

 Time for computing density & RDG =   0.70 secs
 24.04% of small boxes are removed.


-----------------------------------------------------
      Operating grid and increments: Grid-4
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  0.09    0.09    0.09
 nx,ny,nz  =  131     123     102

 Time for computing density & RDG =  33.91 secs
 77.63% of small boxes are removed.
 Time for writing outputs =   2.26 secs
 97.34% of small boxes removed for density integration
                                                     
----------------------------------------------------------------------
                                                    
                     INTEGRATION DATA                        
                                                     
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        1.09128161
 n=1.5           :        0.21644707
 n=2.0           :        0.04410649
 n=2.5           :        0.00912721
 n=3.0           :        0.00190990
 n=4/3           :        0.37007210
 n=5/3           :        0.12736311
 Volume          :       35.93319859
 rho-sum_i rho_i :        0.00043666
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.99542926
 n=1.5           :        0.20447659
 n=2.0           :        0.04167774
 n=2.5           :        0.00862251
 n=3.0           :        0.00180371
 n=4/3           :        0.34947441
 n=5/3           :        0.12034229
----------------------------------------------------------------------
 Integration  over the areas of rho^n              
----------------------------------------------------------------------
 n=1.0           :        9.16655290
 n=1.5           :        1.87831054
 n=2.0           :        0.39145417
 n=2.5           :        0.08237411
 n=3.0           :        0.01745833
 n=4/3           :        3.17803064
 n=5/3           :        1.11211102
 Area            :      261.67872458
 rho-sum_i rho_i :        0.00387544
----------------------------------------------------------------------
                   

 Time for integration =   0.36 secs
                                                                      
----------------------------------------------------------------------
                                                                      
               RANGE INTEGRATION DATA                                 
----------------------------------------------------------------------
                                                                      
#                     Warning!                           #
#       Criterion for active box has changed.            #
# A box is defined as active if ALL its vertices are     #
# within the density and reduced density gradient range. #
#                                                        # 
#                                                        # 
                                                         
                                                         
                                                         
100.00% of small boxes removed for integration of interval 1
----------------------------------------------------------------------
 Interval        :       -0.10000000       -0.01000000                     
                               
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        0.00000000
 n=1.5           :        0.00000000
 n=2.0           :        0.00000000
 n=2.5           :        0.00000000
 n=3.0           :        0.00000000
 n=4/3           :        0.00000000
 n=5/3           :        0.00000000
 Volume          :        0.00000000
 rho-sum_i rho_i :        0.00000000
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.00000000
 n=1.5           :        0.00000000
 n=2.0           :        0.00000000
 n=2.5           :        0.00000000
 n=3.0           :        0.00000000
 n=4/3           :        0.00000000
 n=5/3           :        0.00000000
---------------------------------------------------------------------
                         

 99.33% of small boxes removed for integration of interval 2
----------------------------------------------------------------------
 Interval        :       -0.01000000        0.01000000                     
                               
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        0.03498077
 n=1.5           :        0.00222027
 n=2.0           :        0.00014313
 n=2.5           :        0.00000937
 n=3.0           :        0.00000062
 n=4/3           :        0.00556659
 n=5/3           :        0.00089198
 Volume          :        9.11868812
 rho-sum_i rho_i :        0.00000142
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.02696018
 n=1.5           :        0.00180893
 n=2.0           :        0.00011663
 n=2.5           :        0.00000765
 n=3.0           :        0.00000051
 n=4/3           :        0.00453728
 n=5/3           :        0.00072659
---------------------------------------------------------------------
                         

 98.32% of small boxes removed for integration of interval 3
----------------------------------------------------------------------
 Interval        :        0.01000000        0.10000000                     
                               
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        0.89682341
 n=1.5           :        0.18183402
 n=2.0           :        0.03729948
 n=2.5           :        0.00773181
 n=3.0           :        0.00161816
 n=4/3           :        0.30942509
 n=5/3           :        0.10733351
 Volume          :       22.74779488
 rho-sum_i rho_i :        0.00036927
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.89682341
 n=1.5           :        0.18183402
 n=2.0           :        0.03729948
 n=2.5           :        0.00773181
 n=3.0           :        0.00161816
 n=4/3           :        0.30942509
 n=5/3           :        0.10733351
---------------------------------------------------------------------
                         

End -- 2020.01.07, 03:49:19.396

