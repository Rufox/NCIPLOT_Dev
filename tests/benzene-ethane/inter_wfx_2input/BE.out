 # ----------------- NCIPLOT ------------------------
 # --- PLOTTING NON COVALENT INTERACTION REGIONS ----
 # ---             E.R. Johnson                  ----
 # ---          J. Contreras-Garcia              ----
 # ----------    Duke University         ------------
 #                                                   
 # ---             A. de la Roza                  ---
 # --------- University of California Merced --------
 #                                                   
 # ---               R. A. Boto                   ---
 # ---                 C. Quan                     --
 # --------  Université Pierre et Marie Curie -------
 # --------------------------------------------------
 # ---              Please cite                  ----
 # --J. Am. Chem. Soc., 2010, 132 (18), pp 6498–6506-
 # --------------------------------------------------
 # --------------------------------------------------
 # ---     Contributions for the wfn properties  ----
 # ---      from H. L. Schmider are acknowledged  ---
 # --------------------------------------------------
 # --------------------------------------------------
 # ---     Contributions for the wfx reader      ----
 # ---      from Dave Arias are acknowledged      ---
 # --------------------------------------------------
 #
 # Start -- 2020.01.07, 04:02:11.460

File: /home/ruben/Documents/software/nciplot-3.0/src/nciplot/dat/h__lda.wfc
WARNING(readgrid): Atomic density file not found
File: /home/ruben/Documents/software/nciplot-3.0/src/nciplot/dat/c__lda.wfc
WARNING(readgrid): Atomic density file not found
-----------------------------------------------------
      INPUT INFORMATION:
-----------------------------------------------------

      MIND YOU
      ONLY ANALYZING INTERMOLECULAR INTERACTIONS     


-----------------------------------------------------
      Calculation details:
-----------------------------------------------------
RHO  THRESHOLD   (au): 0.50
RDG  THRESHOLD   (au): 1.00
DISCARDING RHO PARAM : 0.95

-----------------------------------------------------
      Writing output in the following units:
-----------------------------------------------------

 Sign(lambda2)xDensity x Reduced Density Gradient    = BE.dat

 Reduced Density Gradient,RDG      = BE-grad.cube
 Sign(lambda2)xDensity,LS          = BE-dens.cube
 VMD script                        = BE.vmd



-----------------------------------------------------
      Operating grid and increments: Grid-1
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  1.13    1.13    1.13
 nx,ny,nz  =   11      11       9

 Time for computing density & RDG =   0.06 secs
  0.00% of small boxes are removed.


-----------------------------------------------------
      Operating grid and increments: Grid-2
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  0.76    0.76    0.76
 nx,ny,nz  =   17      16      13

 Time for computing density & RDG =   0.09 secs
  3.75% of small boxes are removed.


-----------------------------------------------------
      Operating grid and increments: Grid-3
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  0.38    0.38    0.38
 nx,ny,nz  =   33      31      26

 Time for computing density & RDG =   0.70 secs
 24.04% of small boxes are removed.


-----------------------------------------------------
      Operating grid and increments: Grid-4
-----------------------------------------------------
 x0,y0,z0  =    -6.1434    -5.8006    -1.4631
 x1,y1,z1  =     6.1433     5.8006     8.1722
 ix,iy,iz  =  0.09    0.09    0.09
 nx,ny,nz  =  131     123     102

 Time for computing density & RDG =  33.12 secs
 77.63% of small boxes are removed.
 Time for writing outputs =   2.26 secs
 97.35% of small boxes removed for density integration
                                                     
----------------------------------------------------------------------
                                                    
                     INTEGRATION DATA                        
                                                     
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        1.08926750
 n=1.5           :        0.21602339
 n=2.0           :        0.04401384
 n=2.5           :        0.00910682
 n=3.0           :        0.00190541
 n=4/3           :        0.36936434
 n=5/3           :        0.12710778
 Volume          :       35.84040898
 rho-sum_i rho_i :        0.00043574
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.98954861
 n=1.5           :        0.20410719
 n=2.0           :        0.04160086
 n=2.5           :        0.00860568
 n=3.0           :        0.00179992
 n=4/3           :        0.34883779
 n=5/3           :        0.12012472
----------------------------------------------------------------------
 Integration  over the areas of rho^n              
----------------------------------------------------------------------
 n=1.0           :        9.16629235
 n=1.5           :        1.87959327
 n=2.0           :        0.39198979
 n=2.5           :        0.08253990
 n=3.0           :        0.01750402
 n=4/3           :        3.17945206
 n=5/3           :        1.11312913
 Area            :      261.29483508
 rho-sum_i rho_i :        0.00388074
----------------------------------------------------------------------
                   

 Time for integration =   0.36 secs
                                                                      
----------------------------------------------------------------------
                                                                      
               RANGE INTEGRATION DATA                                 
----------------------------------------------------------------------
                                                                      
#                     Warning!                           #
#       Criterion for active box has changed.            #
# A box is defined as active if ALL its vertices are     #
# within the density and reduced density gradient range. #
#                                                        # 
#                                                        # 
                                                         
                                                         
                                                         
100.00% of small boxes removed for integration of interval 1
----------------------------------------------------------------------
 Interval        :       -0.10000000       -0.01000000                     
                               
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        0.00000000
 n=1.5           :        0.00000000
 n=2.0           :        0.00000000
 n=2.5           :        0.00000000
 n=3.0           :        0.00000000
 n=4/3           :        0.00000000
 n=5/3           :        0.00000000
 Volume          :        0.00000000
 rho-sum_i rho_i :        0.00000000
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.00000000
 n=1.5           :        0.00000000
 n=2.0           :        0.00000000
 n=2.5           :        0.00000000
 n=3.0           :        0.00000000
 n=4/3           :        0.00000000
 n=5/3           :        0.00000000
---------------------------------------------------------------------
                         

 99.33% of small boxes removed for integration of interval 2
----------------------------------------------------------------------
 Interval        :       -0.01000000        0.01000000                     
                               
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        0.03482009
 n=1.5           :        0.00221233
 n=2.0           :        0.00014277
 n=2.5           :        0.00000936
 n=3.0           :        0.00000062
 n=4/3           :        0.00554474
 n=5/3           :        0.00088911
 Volume          :        9.06048373
 rho-sum_i rho_i :        0.00000142
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.02665361
 n=1.5           :        0.00178361
 n=2.0           :        0.00011493
 n=2.5           :        0.00000754
 n=3.0           :        0.00000050
 n=4/3           :        0.00447494
 n=5/3           :        0.00071626
---------------------------------------------------------------------
                         

 98.33% of small boxes removed for integration of interval 3
----------------------------------------------------------------------
 Interval        :        0.01000000        0.10000000                     
                               
----------------------------------------------------------------------
 Integration  over the volumes of rho^n                               
----------------------------------------------------------------------
 n=1.0           :        0.88710141
 n=1.5           :        0.17961944
 n=2.0           :        0.03679622
 n=2.5           :        0.00761756
 n=3.0           :        0.00159222
 n=4/3           :        0.30579422
 n=5/3           :        0.10597913
 Volume          :       22.56137212
 rho-sum_i rho_i :        0.00036428
                  
---------------------------------------------------------------------
 Integration  over the volumes of sign(lambda2)(rho)^n             
---------------------------------------------------------------------
 n=1.0           :        0.88710141
 n=1.5           :        0.17961944
 n=2.0           :        0.03679622
 n=2.5           :        0.00761756
 n=3.0           :        0.00159222
 n=4/3           :        0.30579422
 n=5/3           :        0.10597913
---------------------------------------------------------------------
                         

End -- 2020.01.07, 04:02:48.446

